### DISCUSSION TITLE
Community outreach and education

### DISCUSSION THREAD

**Discussion by samikanza** [MADICES/MADICES-2025#4] [Ideas]
**MADICES 2024 Recap**
At MADICES we want to develop educational resources and workshops aimed at broadening adoption and understanding of open science practices. For example, the development of the platform-agnostic interoperability web app demonstrator has highlighted its value as a learning platform and is presently considered for an online-based playground allowing users to try the interoperability tools and protocols live.

**MADICES 2025 Focus Area**
This remains a key focus area for MADICES 2025 with a look to bring our diverse set of stakeholders together to identify further tools, resources and workshops for the community going forward, and to collectively identify any new tools/resources that have been developed since the last workshop to add to our tools list. 

Please include links, comments, and discuss plans for this focus area below.

**Comment by budschi** [comment]
There is a huge gap between people like us going to this MADICES workshop here and people working in the labs just wanting to get their research done. How do we better bridge this gap? We probably need help in defining good Data Governance Policies that research organisation actually also want to follow and practive actively. Key point is lack of defined roles in research data management which lead to overwhelmed or badly organized practices at institutes.

**Comment by hampusnasstrom** [comment]
I think it would be good to also compile a list of resources on how to (in practice!) apply semantics to your data. Maybe making an [awesome-list](https://github.com/topics/awesome-list) could be a task for the workshop?

**Reply by ml-evs** [reply]
Before MADICES, there was a thing we were calling "neo-chem" which started this process at https://github.com/neo-chem/awesome-chemical-data -- it ended up being hosted under MADICES here: https://madices.github.io/docs/awesome_interoperability 

Agree that there's limited concrete advice though, perhaps because even amongst us there are very few exemplar cases

**Reply by ml-evs** [reply]
We also have an empty repo called "The Pauling Way" a la [The Turing Way](https://the-turing-way.start.page/) which we should have done something with for the name alone :upside_down_face:, but a guide like that, whilst a huge amount of effort, would be very valuable IMO.

**Comment by openbis** [comment]
Discussion documented in a google doc: https://docs.google.com/document/d/1Q_qxbaP2cJg4V60BNi5Z-AEv3tTyMgxmwo8gzfxM64M/edit?tab=t.0 

Slides
https://docs.google.com/presentation/d/18DW6bCnYYxKj_CGrXmCXTKW_vMe-TE1sgMplgXFdc9c/edit?slide=id.p#slide=id.p

**Comment by davidelbert** [comment]
This topic is, of course, central to the mission of MaRDA and something we have been working on for quite a while.  https://marda-alliance.org.  While MaRDA started in the US it is really fairly international and if any of you want a home to connect the diverse stakeholders I invite anyone to suggest a working group there.  We explicitly work with stakeholders from: academia, industry research, instrument manufacturing, repositories, digital libraries/librarians, publishing, PID communities, national laboratories, and higher-level policy organizations like funders and government agencies.  A common finding for us is that the stakeholders have different missions, different funding implications, and very different skill sets/awareness.  Another common finding is that the communities don't interact enough to understand and respect each other's needs and prior work.  My experience is to go to meetings in each of these communities and hear frustrations about something the other stakeholders are doing.  It makes me firmly belief (pure opinion, I admit) that creating tools and workshops to "train" others to do the right thing rarely moves the needle.  I will provide a somewhat creative (I think!) approach one of our MaRDA groups has taken in another comment.
